artio-codecs/src/main/java/uk/co/real_logic/artio/dictionary/CodecGenerationTool.java

```diff
@@ -31,6 +31,8 @@
 
 public final class CodecGenerationTool
 {
+    public static final String VALIDATION_ALLOW_DUPLICATES = "fix.validation.allow.duplicates";
```

Comment:
The format for our other validation properties is `fix.codecs.some_descriptive_name`, so this should be `fix.codecs.allow_duplicate_fields`

Reply:
done